pkgs/applications/editors/emacs/macport.nix

```diff
@@ -33,17 +33,19 @@ stdenv.mkDerivation rec {
   buildInputs = [ ncurses libxml2 gnutls texinfo gettext jansson
     AppKit Carbon Cocoa IOKit OSAKit Quartz QuartzCore WebKit
     ImageCaptureCore GSS ImageIO   # may be optional
-  ];
+  ] ++ lib.optionals stdenv.isAarch64 [ UniformTypeIdentifiers sigtool ];
```

Comment:
sigtool is executed on the build machine, right? Then it needs to go to nativeBuildInputs.

Reply:
@Kloenk please move this into nativeBuildInputs.